test/util/testnode/full_node.go

```diff
@@ -147,13 +148,17 @@ func DefaultTendermintConfig() *config.Config {
 	// during tests.
 	tmCfg.Consensus.TargetHeightDuration = 300 * time.Millisecond
 	tmCfg.Consensus.TimeoutPropose = 200 * time.Millisecond
-	tmCfg.Mempool.MaxTxBytes = 22020096 // 21MB
+	// remove all barriers from the testnode being able to accept very large
+	// transactions (max square size) and respond to very large queries (200MiB)
+	tmCfg.Mempool.MaxTxBytes = appconsts.MaxShareCount * appconsts.ShareSize
+	tmCfg.RPC.MaxBodyBytes = 200_000_000
```

Comment:
thanks for not blocking, for posterity what do I need in order to say 2 hundred million Bytes in shortform? 2e8 Bytes? ~200MB? unless the precision is actually called for, I would really like to avoid having to use a website or a calculator to write comments for tests :slightly_smiling_face: 

this is why I thought it was common place to implicitly switch to the metric system's "Mega" in almost all casual discussions/values.
![Screenshot from 2023-05-18 11-26-07](https://github.com/celestiaorg/celestia-app/assets/42654277/7f3e60e7-88dd-48a1-a6da-cb5e2d808dca)


> to include the godoc for a line immediately above

the change makes sense, so I'll change that in a followup or I'll toss it in if there are other requested changes :slightly_smiling_face:. wait, does this show up on a godoc? I thought this was just a line for anyone curious why we set these values in the tests. 

> I don't understand what is the rationale for testing "very large queries"

Literally it was just because if transaction or repsponses to those transactions exceeds that value it errors, to avoid that in general it was increased. It could be applied to any interation with the node, we mainly use it for tx or getting large blocks

Reply:
okay went ahead and hopefully fixed it as we should wait for Callum's feedback anyway, pls let me know if we should update this still or there is any clarification needed

6d564db30403e85a14da850497293dccb74eb902

Reply:
> what do I need in order to say 2 hundred million Bytes in shortform? 2e8 Bytes? ~200MB?

`200 MB` works so I think this could've been either:

```
tmCfg.RPC.MaxBodyBytes = 200_000_000 // 200 MB
tmCfg.RPC.MaxBodyBytes = 209_715_200 // 200 MiB
```

Note: MiB is [mebibyte](https://www.techtarget.com/searchstorage/definition/mebibyte-MiB) which is different from megabyte.

> wait, does this show up on a godoc?

Sorry, I guess it doesn't show up in a godoc. I just meant that it may help readers if the comment is applicable to the line immediately below it.

Reply:
> Note: MiB is [mebibyte](https://www.techtarget.com/searchstorage/definition/mebibyte-MiB) which is different from megabyte.

this is what I thought MB and MiB were

I was confused because of [this](https://github.com/celestiaorg/celestia-app/pull/1772#discussion_r1196854199), which uses MB

Reply:
so I just switched to MiB cause I thought you were refering to both interchangably